spec/factories/target_factory.rb

```diff
@@ -0,0 +1,17 @@
+# This will guess the User class
+FactoryBot.define do
+  factory :target do
+    topic 'Football'
+    length '200'
+    title 'Prueba'
+    latitude '-34'
+    longitude '-54'
+  end
+
+  # This will use the User class (Admin would have been guessed)
```

Comment:
Is this commented code being included later? I guess yes, but if not please erase it